jsfiles/index.js

```diff
@@ -1,16 +1,277 @@
+	// 				Technologies: ['html', 'css', 'JavaScript'],
+	// 				projectText: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.",
+	// 				liveLink: '#',
+	// 				sourceLink: '#',
+	// 		},
+	// 		{
+	// 				key: 2,
+	// 				Name: 'Tonic',
+	// 				Image: './images/tonicProject2.png',
+	// 				companyName: 'CANOPY',
+	// 				projectPosition: 'Back And Dev',
+	// 				projectDate: 2015,
+	// 				Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+	// 				Technologies: ['html', 'css', 'JavaScript'],
+	// 				projectText: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.",
+	// 				liveLink: '#',
+	// 				sourceLink: '#',
+	// 		},
+	// 		{
+	// 				key: 3,
+	// 				Name: 'Multi-Post Stories',
+	// 				Image: './images/multiPostProject2.png',
+	// 				companyName: 'CANOPY',
+	// 				projectPosition: 'Back And Dev',
+	// 				Date: 2015,
+	// 				Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+	// 				Technologies: ['html', 'css', 'JavaScript'],
+	// 				projectText: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.",
+	// 				liveLink: '#',
+	// 				sourceLink: '#',
+	// 		},
+	// ];
+
+
+
+// console.log(projects);
+
+// /Volumes/local storage/Microverse/M-portfolio/Assets/images/Portfolio1.png
+
+// M-portfolio/Assets/images/Portfolio1.png
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+// let messsage = [];
+// projectDetails = (e) => {
+// 	e.preventDefault
+// 	const popUpText = document.createElement('p');
+// }
+
+// const seeProject = document.querySelectorAll('.project');
+// for (let i = 0; i < seeProject.length; i++) {
+// 	seeProject[i].addEventListener('click', projectDetails);
+
+// }
+// const mobileProjects = document.querySelector('.project-works');
+// const desktopProjects = document.querySelector('.grid-container')
+
+// const projects = [mobileProjects, desktopProjects];
+// for (let i = 0; i < projects.length; i++){
+// 	console.log(projects[i]);
+// }
+
+
```

Comment:
- [ ] [OPTIONAL] I'm gonna put this as optional, but for future projects, all commented code must be gone, it's unnecesary.

Reply:
I forgot to delete that. I used it this morning for the code review session when I was explaining the code to my morning session group.